APE0.rst

```diff
@@ -157,7 +157,7 @@ In the case of a vacancy partway through a term, a by-election will be held.
 The term of the newly-elected member runs for the remainder of the term of
 the member they are replacing. 
 
-For the initial election of Coordination Committee members, two seats will have
+For the initial election of Coordination Committee members (four at the time instead of five), two seats will have
 terms of one year, one has a term of two years, and one has a term of three
 years to create a staggered set of replacements and provide continuity in the
```

Comment:
This paragraph determines the number of new members that will be elected to the CoCo in future elections, so it is important that it explicitly states the terms of all five initial members of the CoCo.

Reply:
That I don't know how to fix. And to ask @astropy/coordinators to figure it out might cause conflict of interest. Hmm...

Reply:
There's an email somewhere that explicitly states our different terms. Not sure where the "conflict" would be.

Reply:
@kelle , do you still have that email? Would be nice to put that officially here, maybe?

Reply:
We did not spec out what would happen for 5 people on the coco, since the original plan was for 4. I think essentially Stuart decided ad hoc, that we do 2 for one year,  2 for two years, and one for three years